build-info-extractor/src/test/java/org/jfrog/build/extractor/clientConfiguration/client/ArtifactoryManagerTest.java

```diff
@@ -51,18 +46,49 @@ protected void cleanup() throws IOException {
      * Send build info to artifactory, receive it and compare.
      */
     @Test
-    public void sendBuildInfoAndBuildRetentioTest() throws IOException {
-        doSendBuildInfoTest(null);
-        sendBuildRetention("");
+    public void sendBuildInfoAndBuildRetentionTest() throws IOException {
+        try {
+            publishAndCheckBuildInfo(null);
+            sendBuildRetention("");
+        } finally {
+            cleanTestBuilds(BUILD_NAME, BUILD_NUMBER, null);
```

Comment:
We send here ```null``` for the projecr argument, while we also send an empty string for this argument on other calls for this method. If null works, we better use it instead of an empty string.